[2024-05-22 02:20:35,261] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T19:00:00+00:00 [queued]>
[2024-05-22 02:20:35,399] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T19:00:00+00:00 [queued]>
[2024-05-22 02:20:35,400] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 02:20:35,403] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2024-05-22 02:20:35,403] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 02:20:35,548] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): fetch_data> on 2024-05-21 19:00:00+00:00
[2024-05-22 02:20:35,648] {standard_task_runner.py:52} INFO - Started process 2399 to run task
[2024-05-22 02:20:35,658] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'purpleair_api_dag', 'fetch_data', 'scheduled__2024-05-21T19:00:00+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion.py', '--cfg-path', '/tmp/tmpjdpznd61', '--error-file', '/tmp/tmpxq890zsb']
[2024-05-22 02:20:35,766] {standard_task_runner.py:77} INFO - Job 20: Subtask fetch_data
[2024-05-22 02:20:36,562] {logging_mixin.py:109} INFO - Running <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T19:00:00+00:00 [running]> on host 8fe6997c1526
[2024-05-22 02:20:37,153] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=purpleair_api_dag
AIRFLOW_CTX_TASK_ID=fetch_data
AIRFLOW_CTX_EXECUTION_DATE=2024-05-21T19:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-21T19:00:00+00:00
[2024-05-22 02:20:37,852] {logging_mixin.py:109} INFO - Failed to retrieve data: 403
[2024-05-22 02:20:37,874] {python.py:175} INFO - Done. Returned value was: None
[2024-05-22 02:20:38,107] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=purpleair_api_dag, task_id=fetch_data, execution_date=20240521T190000, start_date=20240522T022035, end_date=20240522T022038
[2024-05-22 02:20:38,351] {local_task_job.py:154} INFO - Task exited with return code 0
[2024-05-22 02:20:38,660] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-22 04:17:53,199] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T19:00:00+00:00 [queued]>
[2024-05-22 04:17:53,352] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T19:00:00+00:00 [queued]>
[2024-05-22 04:17:53,353] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 04:17:53,354] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2024-05-22 04:17:53,355] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 04:17:53,495] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): fetch_data> on 2024-05-21 19:00:00+00:00
[2024-05-22 04:17:53,518] {standard_task_runner.py:52} INFO - Started process 381 to run task
[2024-05-22 04:17:53,567] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'purpleair_api_dag', 'fetch_data', 'scheduled__2024-05-21T19:00:00+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion.py', '--cfg-path', '/tmp/tmpukz4ox4f', '--error-file', '/tmp/tmpw6xihgw_']
[2024-05-22 04:17:53,636] {standard_task_runner.py:77} INFO - Job 39: Subtask fetch_data
[2024-05-22 04:17:54,397] {logging_mixin.py:109} INFO - Running <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T19:00:00+00:00 [running]> on host 3419eee920a5
[2024-05-22 04:17:54,737] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=purpleair_api_dag
AIRFLOW_CTX_TASK_ID=fetch_data
AIRFLOW_CTX_EXECUTION_DATE=2024-05-21T19:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-21T19:00:00+00:00
[2024-05-22 04:17:54,741] {logging_mixin.py:109} INFO -  api is None
[2024-05-22 04:17:54,776] {logging_mixin.py:109} INFO - Full URL: https://api.purpleair.com/v1/sensors?fields=latitude%2Clongitude%2Chumidity%2Cpm2.5_60minute%2Clast_seen&location_type=0&max_age=300
[2024-05-22 04:17:54,777] {logging_mixin.py:109} INFO - Headers: {'X-API-Key': 'None'}
[2024-05-22 04:17:54,946] {logging_mixin.py:109} INFO - Status Code: 403
[2024-05-22 04:17:54,947] {logging_mixin.py:109} INFO - Response Content: {
  "api_version" : "V1.0.13-0.0.56",
  "time_stamp" : 1716351474,
  "error" : "ApiKeyInvalidError",
  "description" : "The provided api_key was not valid."
}
[2024-05-22 04:17:55,026] {logging_mixin.py:109} INFO - Failed to retrieve data: 403
[2024-05-22 04:17:55,027] {python.py:175} INFO - Done. Returned value was: None
[2024-05-22 04:17:55,064] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=purpleair_api_dag, task_id=fetch_data, execution_date=20240521T190000, start_date=20240522T041753, end_date=20240522T041755
[2024-05-22 04:17:55,186] {local_task_job.py:154} INFO - Task exited with return code 0
[2024-05-22 04:17:55,479] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-22 06:56:30,672] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T19:00:00+00:00 [queued]>
[2024-05-22 06:56:30,893] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T19:00:00+00:00 [queued]>
[2024-05-22 06:56:30,908] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 06:56:30,912] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2024-05-22 06:56:30,924] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 06:56:31,057] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): fetch_data> on 2024-05-21 19:00:00+00:00
[2024-05-22 06:56:31,106] {standard_task_runner.py:52} INFO - Started process 566 to run task
[2024-05-22 06:56:31,159] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'purpleair_api_dag', 'fetch_data', 'scheduled__2024-05-21T19:00:00+00:00', '--job-id', '56', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion.py', '--cfg-path', '/tmp/tmprizs9ncm', '--error-file', '/tmp/tmpq05dr0_v']
[2024-05-22 06:56:31,237] {standard_task_runner.py:77} INFO - Job 56: Subtask fetch_data
[2024-05-22 06:56:31,619] {logging_mixin.py:109} INFO - Running <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T19:00:00+00:00 [running]> on host 148b4063f71c
[2024-05-22 06:56:31,973] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=purpleair_api_dag
AIRFLOW_CTX_TASK_ID=fetch_data
AIRFLOW_CTX_EXECUTION_DATE=2024-05-21T19:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-21T19:00:00+00:00
[2024-05-22 06:56:32,031] {logging_mixin.py:109} INFO - Full URL: https://api.purpleair.com/v1/sensors?fields=latitude%2Clongitude%2Chumidity%2Cpm2.5_60minute%2Ctemperature&location_type=0&max_age=300
[2024-05-22 06:56:32,052] {logging_mixin.py:109} INFO - Headers: {'X-API-Key': '9A23CD90-1279-11EF-B9F7-42010A80000D'}
[2024-05-22 06:56:32,196] {logging_mixin.py:109} INFO - Status Code: 402
[2024-05-22 06:56:32,246] {logging_mixin.py:109} INFO - Response Content: {
  "api_version" : "V1.0.13-0.0.56",
  "time_stamp" : 1716360992,
  "error" : "PaymentRequiredError",
  "description" : "Payment is required to make this api call."
}
[2024-05-22 06:56:32,449] {logging_mixin.py:109} INFO - Failed to retrieve data: 402
[2024-05-22 06:56:32,450] {python.py:175} INFO - Done. Returned value was: None
[2024-05-22 06:56:32,930] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=purpleair_api_dag, task_id=fetch_data, execution_date=20240521T190000, start_date=20240522T065630, end_date=20240522T065632
[2024-05-22 06:56:33,117] {local_task_job.py:154} INFO - Task exited with return code 0
[2024-05-22 06:56:33,363] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-22 07:04:50,533] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T19:00:00+00:00 [queued]>
[2024-05-22 07:04:50,628] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T19:00:00+00:00 [queued]>
[2024-05-22 07:04:50,629] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 07:04:50,636] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2024-05-22 07:04:50,637] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 07:04:50,719] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): fetch_data> on 2024-05-21 19:00:00+00:00
[2024-05-22 07:04:50,741] {standard_task_runner.py:52} INFO - Started process 1106 to run task
[2024-05-22 07:04:50,784] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'purpleair_api_dag', 'fetch_data', 'scheduled__2024-05-21T19:00:00+00:00', '--job-id', '134', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion.py', '--cfg-path', '/tmp/tmpmtwob1gb', '--error-file', '/tmp/tmp0s96vht9']
[2024-05-22 07:04:50,815] {standard_task_runner.py:77} INFO - Job 134: Subtask fetch_data
[2024-05-22 07:04:51,087] {logging_mixin.py:109} INFO - Running <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T19:00:00+00:00 [running]> on host 148b4063f71c
[2024-05-22 07:04:51,341] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=purpleair_api_dag
AIRFLOW_CTX_TASK_ID=fetch_data
AIRFLOW_CTX_EXECUTION_DATE=2024-05-21T19:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-21T19:00:00+00:00
[2024-05-22 07:04:51,354] {logging_mixin.py:109} INFO - Full URL: https://api.purpleair.com/v1/sensors?fields=latitude%2Clongitude%2Chumidity%2Cpm2.5_60minute%2Ctemperature&location_type=0&max_age=300
[2024-05-22 07:04:51,372] {logging_mixin.py:109} INFO - Headers: {'X-API-Key': '9A23CD90-1279-11EF-B9F7-42010A80000D'}
[2024-05-22 07:04:51,556] {logging_mixin.py:109} INFO - Status Code: 402
[2024-05-22 07:04:51,557] {logging_mixin.py:109} INFO - Response Content: {
  "api_version" : "V1.0.13-0.0.56",
  "time_stamp" : 1716361491,
  "error" : "PaymentRequiredError",
  "description" : "Payment is required to make this api call."
}
[2024-05-22 07:04:51,774] {logging_mixin.py:109} INFO - Failed to retrieve data: 402
[2024-05-22 07:04:51,775] {python.py:175} INFO - Done. Returned value was: None
[2024-05-22 07:04:51,839] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=purpleair_api_dag, task_id=fetch_data, execution_date=20240521T190000, start_date=20240522T070450, end_date=20240522T070451
[2024-05-22 07:04:52,058] {local_task_job.py:154} INFO - Task exited with return code 0
[2024-05-22 07:04:52,392] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-22 07:12:41,697] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T19:00:00+00:00 [queued]>
[2024-05-22 07:12:41,792] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T19:00:00+00:00 [queued]>
[2024-05-22 07:12:41,793] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 07:12:41,793] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2024-05-22 07:12:41,793] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 07:12:41,909] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): fetch_data> on 2024-05-21 19:00:00+00:00
[2024-05-22 07:12:41,948] {standard_task_runner.py:52} INFO - Started process 1638 to run task
[2024-05-22 07:12:41,966] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'purpleair_api_dag', 'fetch_data', 'scheduled__2024-05-21T19:00:00+00:00', '--job-id', '215', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion.py', '--cfg-path', '/tmp/tmpq0ppmmm4', '--error-file', '/tmp/tmpedna2wvs']
[2024-05-22 07:12:41,997] {standard_task_runner.py:77} INFO - Job 215: Subtask fetch_data
[2024-05-22 07:12:42,215] {logging_mixin.py:109} INFO - Running <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T19:00:00+00:00 [running]> on host 148b4063f71c
[2024-05-22 07:12:42,453] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=purpleair_api_dag
AIRFLOW_CTX_TASK_ID=fetch_data
AIRFLOW_CTX_EXECUTION_DATE=2024-05-21T19:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-21T19:00:00+00:00
[2024-05-22 07:12:42,470] {logging_mixin.py:109} INFO - Full URL: https://api.purpleair.com/v1/sensors?fields=latitude%2Clongitude%2Chumidity%2Cpm2.5_60minute%2Ctemperature&location_type=0&max_age=300
[2024-05-22 07:12:42,471] {logging_mixin.py:109} INFO - Headers: {'X-API-Key': '9A23CD90-1279-11EF-B9F7-42010A80000D'}
[2024-05-22 07:12:42,652] {logging_mixin.py:109} INFO - Status Code: 402
[2024-05-22 07:12:42,653] {logging_mixin.py:109} INFO - Response Content: {
  "api_version" : "V1.0.13-0.0.56",
  "time_stamp" : 1716361962,
  "error" : "PaymentRequiredError",
  "description" : "Payment is required to make this api call."
}
[2024-05-22 07:12:42,778] {logging_mixin.py:109} INFO - Failed to retrieve data: 402
[2024-05-22 07:12:42,779] {python.py:175} INFO - Done. Returned value was: None
[2024-05-22 07:12:43,056] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=purpleair_api_dag, task_id=fetch_data, execution_date=20240521T190000, start_date=20240522T071241, end_date=20240522T071243
[2024-05-22 07:12:43,212] {local_task_job.py:154} INFO - Task exited with return code 0
[2024-05-22 07:12:43,864] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-22 07:22:24,837] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T19:00:00+00:00 [queued]>
[2024-05-22 07:22:24,925] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T19:00:00+00:00 [queued]>
[2024-05-22 07:22:24,925] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 07:22:24,926] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2024-05-22 07:22:24,926] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 07:22:25,085] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): fetch_data> on 2024-05-21 19:00:00+00:00
[2024-05-22 07:22:25,134] {standard_task_runner.py:52} INFO - Started process 2344 to run task
[2024-05-22 07:22:25,199] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'purpleair_api_dag', 'fetch_data', 'scheduled__2024-05-21T19:00:00+00:00', '--job-id', '325', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion.py', '--cfg-path', '/tmp/tmpcwhojmb9', '--error-file', '/tmp/tmpl02bf1uu']
[2024-05-22 07:22:25,255] {standard_task_runner.py:77} INFO - Job 325: Subtask fetch_data
[2024-05-22 07:22:25,542] {logging_mixin.py:109} INFO - Running <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T19:00:00+00:00 [running]> on host 148b4063f71c
[2024-05-22 07:22:25,701] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=purpleair_api_dag
AIRFLOW_CTX_TASK_ID=fetch_data
AIRFLOW_CTX_EXECUTION_DATE=2024-05-21T19:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-21T19:00:00+00:00
[2024-05-22 07:22:25,726] {logging_mixin.py:109} INFO - Full URL: https://api.purpleair.com/v1/sensors?fields=latitude%2Clongitude%2Chumidity%2Cpm2.5_60minute%2Ctemperature&location_type=0&max_age=300
[2024-05-22 07:22:25,726] {logging_mixin.py:109} INFO - Headers: {'X-API-Key': '9A23CD90-1279-11EF-B9F7-42010A80000D'}
[2024-05-22 07:22:25,992] {logging_mixin.py:109} INFO - Status Code: 402
[2024-05-22 07:22:26,010] {logging_mixin.py:109} INFO - Response Content: {
  "api_version" : "V1.0.13-0.0.56",
  "time_stamp" : 1716362545,
  "error" : "PaymentRequiredError",
  "description" : "Payment is required to make this api call."
}
[2024-05-22 07:22:26,525] {logging_mixin.py:109} INFO - Failed to retrieve data: 402
[2024-05-22 07:22:26,526] {python.py:175} INFO - Done. Returned value was: None
[2024-05-22 07:22:26,634] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=purpleair_api_dag, task_id=fetch_data, execution_date=20240521T190000, start_date=20240522T072224, end_date=20240522T072226
[2024-05-22 07:22:26,835] {local_task_job.py:154} INFO - Task exited with return code 0
[2024-05-22 07:22:27,586] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
